dynawo/sources/Models/Modelica/Dynawo/Examples/Wind/IEC/IECWT4ANeplan/WT4BCurrentSource.mo

```diff
@@ -0,0 +1,104 @@
+within Dynawo.Examples.Wind.IEC.IECWT4ANeplan;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model WT4BCurrentSource "Wind Turbine Type 4B model from IEC 61400-27-1 standard with infinite bus - fault and reference tracking tests (Active and reactive power steps)"
+  import Modelica;
+  import Dynawo;
+
+  extends Icons.Example;
+
+  Dynawo.Electrical.Wind.IEC.WT4BCurrentSource wT4BCurrentSource(BesPu = 0, CdrtPu = 15, DPMaxP4BPu = 1, DPRefMax4BPu = 100, DPRefMin4BPu = -100, DUdb1Pu = -0.1, DUdb2Pu = 0.1, DfMaxPu = 1, DipMaxPu = 1, DiqMaxPu = 100, DiqMinPu = -100, GesPu = 0, Hgen = 1, Hwtr = 5, IGsIm0Pu = 0.423168, IGsRe0Pu = 0.930069, IMaxDipPu = 1.3, IMaxPu = 1.3, IpMax0Pu = 1.2, IqH1Pu = 1.05, IqMax0Pu = 0.4, IqMaxPu = 1.05, IqMin0Pu = -0.4, IqMinPu = -1.05, IqPostPu = 0, KdrtPu = 500, Kipaw = 100, Kiq = 2.25, Kiqaw = 100, Kiu = 10, Kpaw = 1000, Kpq = 1.1, Kpqu = 20, Kpu = 2, Kpufrt = 2, Kqv = 2, MdfsLim = false, MpUScale = false, MqG = 1, Mqfrt = 1, Mqpri = true, P0Pu = -1, PAg0Pu = 1, Q0Pu = 0.21, QMax0Pu = 0.8, QMaxPu = 0.8, QMin0Pu = -0.8, QMinPu = -0.8, QlConst = true, RDropPu = 0, ResPu = 0, SNom = 100, U0Pu = 1, UGsIm0Pu = 0.21823, UGsRe0Pu = 0.975897, UMaxPu = 1.1, UMinPu = 0.9, UOverPu = 1.1, UPhase0 = 0.21949, UPll1Pu = 999, UPll2Pu = 0.13, URef0Pu = 0, UUnderPu = 0.9, UpDipPu = 0, UpquMaxPu = 1.1, UqDipPu = 0.9, UqRisePu = 1.1, XDropPu = 0, XWT0Pu = -0.21, XesPu = 0, fOverPu = 1.1, fUnderPu = 0.9, i0Pu = Complex(-0.930069, -0.423168), tG = 0.01, tIFilt = 0.01, tPAero = 0.1, tPFilt = 0.01, tPOrdP4B = 0.05, tPll = 0.01, tPost = 0, tQFilt = 0.01, tQord = 0.05, tS = 0.001, tUFilt = 0.01, tUss = 1, tfFilt = 0.01, u0Pu = Complex(0.975897, 0.2183))  annotation(
+    Placement(visible = true, transformation(origin = {-130, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Dynawo.Electrical.Events.NodeFault nodeFault1(RPu = 0, XPu = 0.215, tBegin = 12, tEnd = 12.15)  annotation(
```

Comment:
Maybe we could simulate exactly the same events and at the same time, between type A and type B